domains/gabrigode.json

```
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "gabrigode",
+           "email": "",
+           "discord": "368920323808034816",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.rlXCntE-RbS7-N1gSETyncs-hRDIJnsaS_BkbzFjW7qS94B4wt34p7EwUyiaeemwbW3EnQVD-z-PXQoxKI2dLZQhJR2LLbuK1aWG1sQZREOT5J_DcjuY1HXpcsnD_zu_s0uZ7Pfs_b5OMrrqTyUB0hEmR72J7Qg4XNFsy5-ges6EboFzHeXYxolBo-g7nsdPgvcw6EVJdoaCvl0OvNMoIfxyOf1GkeEBAXxAfutsWLLYp4U7HN1a6AkOSPLVkgIwHgi8lGIHWJjYqgjfZd8nRTIe_4d9pOJJVlKVfPp9i8fsl1oysFEVefiU_Xx6AlfXSm2UZcbmMwRq3ItzMxqviA.Zpx8XSXQNPjcWVDZNJuH2Q.LrGHXkueDq4PQp_ptTfE7-2dnJUJ5P86ny7dQkSu4qApFr-X32Sxm5wAG7zbOH_bVakgYvL-Va-5sKS5UHIbU0Ne5rK4CvJ0HxEzLDG8fpA.ilZ6z0z2oCEI7yP2cHdVOQ"
+        },
+    
+        "record": {
+            "CNAME": "github.com/gabrigode"
```

CNAMEs cannot contain slashes `/`. If you want this to redirect to your Github profile, I recommend using https://redirect.pizza and following their instructions; or you can use our (unstable) URL records like this:

```suggestion
            "URL": "https://github.com/gabrigode"
```